### ISSUE TITLE
Python Django package

### ISSUE THREAD

**Issue by jonashaag** [FGRibreau/mailchecker#136]
I made a tiny wrapper for Django (Python web framework). Is there any place I can put a link to the repo? Maybe in the README?

https://github.com/jonashaag/django-indisposable

**Comment by FGRibreau** [comment]
Yes, you could add a section after "MailChecker currently supports:", like "MailChecker wrappers for frameworks": and then add a link to django-indisposable, what do you think?
